src/internal/kopia/wrapper.go

```diff
@@ -495,8 +495,17 @@ func inflateDirTree(
 	return res, ownerCats, nil
 }
 
+// BackupCollections takes a set of collections and creates a kopia snapshot
+// with the data that they contain. previousSnapshots is used for incremental
+// backups and should represent the base snapshot from which metadata is sourced
+// from as well as any incomplete snapshot checkpoints that may contain more
+// recent data than the base snapshot. Passing nil for previousSnapshots means
+// denotes that a complete backup of all data should be done.
```

Comment:
Minor: I'm not a fan of specifying `nil` here, as that sidesteps what should be equal behavior for an empty collection.  And I wouldn't want to encourage producer funcs to return `nil, nil` as a standard.
```suggestion
// recent data than the base snapshot. Passing nil for previousSnapshots means
// denotes that a complete backup of all data should be done.
```
```suggestion
// recent data than the base snapshot. The absence of previousSnapshots
// causes a complete backup of all data.
```

Reply:
fair enough. I wasn't quite sure what to put cause "nil or empty collection" sounds a bit wordy but just one or the other has the issues you pointed out 🙃